logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/CreateOneToOneConversationUseCase.kt

```diff
@@ -0,0 +1,34 @@
+package com.wire.kalium.logic.feature.conversation
+
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.data.conversation.ConversationRepository
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.user.UserId
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.functional.suspending
+
+class CreateOneToOneConversationUseCase(
```

Comment:
```suggestion
class GetOrCreateOneToOneConversationUseCase(
```

It's a bit clearer what it does.

We can call it multiple times without being afraid of creating multiple 1:1 conversations. Also, if some app actually wants to create multiple conversations, the name would clearly indicate why it doesn't work.